2.2.2/bullseye/build-time-pip-constraints.txt

```diff
@@ -27,27 +27,28 @@ PyYAML==5.4.1
 Pygments==2.10.0
 Rx==3.2.0
 SQLAlchemy-JSONField==1.0.0
-SQLAlchemy-Utils==0.37.8
+SQLAlchemy-Utils==0.37.9
 SQLAlchemy==1.3.24
 SecretStorage==3.3.1
 Sphinx==3.4.3
 Unidecode==1.3.2
 WTForms==2.3.3
 Werkzeug==1.0.1
 adal==1.2.7
-aiohttp==3.7.4.post0
+aiohttp==3.8.0
+aiosignal==1.2.0
 alabaster==0.7.12
 alembic==1.7.4
 aliyun-python-sdk-core==2.13.35
 aliyun-python-sdk-kms==2.15.0
 amqp==5.0.6
 analytics-python==1.4.0
 ansiwrap==0.8.4
-anyio==3.3.2
+anyio==3.3.4
 apache-airflow-providers-airbyte==1!2.1.1
 apache-airflow-providers-alibaba==1!1.0.0
-apache-airflow-providers-amazon==1!2.2.0
-apache-airflow-providers-apache-beam==1!3.0.1
+apache-airflow-providers-amazon==1!2.3.0
```

Comment:
@kaxil, I don't think these have been built yet, but I'm not sure how to do that.

Reply:
Doing that now